INSTALL.md

```diff
@@ -181,7 +181,7 @@ github organization. Each of the plugins have their own installation instruction
 suggest looking at each individual .travis.yml file to see detailed installation
 instructions. Note that the most common plugins are:
 - [qtp-biom](https://github.com/qiita-spots/qtp-biom)
-- [qtp-target-gene](https://github.com/qiita-spots/qtp-target-gene)
+- [qtp-sequencing](https://github.com/qiita-spots/qtp-sequencing)
```

Comment:
qtp-target-gene has been renamed to qtp-sequencing. The old link still works, but it is surprising (and potentially worrying :) to the reader to end up at a different URL than the one they expected.